src/components/DisplayRandom.js

```diff
@@ -0,0 +1,38 @@
+function DisplayRandom (props) {
+    const buttonClicked = props.buttonClick;
+    if (buttonClicked) {
+    return (
+        <section className="wrapper">
+            <div className="queen">
+
+                <div className="queenLeft">
+                    <div className="imgContainer">
+                    {props.buttonClick && <img src={props.randomQueen.image_url} alt={`${props.randomQueen.name}`}/>}
```

Comment:
From line 10 to 27, to clean up the code a bit, you can try destructuring props. This can also apply to other components where you can destructure.